app/src/main/java/com/netmera/netmerafintech/utils/Extensions.kt

```diff
@@ -8,7 +8,14 @@ import android.app.Activity
 import android.os.Build.VERSION.SDK_INT
 import android.os.Bundle
 import android.os.Parcelable
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.view.inputmethod.EditorInfo
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
 import android.widget.Toast
+import androidx.fragment.app.Fragment
 
 fun Activity.toast(message: String) {
     Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
```

Comment:
Adding toast length as a parameter would be nice. And the default value should be LONG because of long texts.